octobot_evaluators/api/evaluators.py

```diff
@@ -32,26 +32,30 @@
 from octobot_evaluators.evaluator.abstract_util import AbstractUtil
 
 EvaluatorClassTypes = {
-    "TA": TAEvaluator,
-    "SOCIAL": SocialEvaluator,
-    "REAL_TIME": RealTimeEvaluator,
-    "STRATEGIES": StrategyEvaluator
+    EvaluatorMatrixTypes.TA.value: TAEvaluator,
+    EvaluatorMatrixTypes.SOCIAL.value: SocialEvaluator,
+    EvaluatorMatrixTypes.REAL_TIME.value: RealTimeEvaluator,
+    EvaluatorMatrixTypes.STRATEGIES.value: StrategyEvaluator
```

Comment:
I don't really understand.

Reply:
It's the same value at the end but using an enum allows for other modules to use this dict more easily (web interface is using it)

Reply:
So you should have remove the concerned enum from the module. We can't keep an enum that correspond to this values. This could be problematic in the future.